vaadin-grid-flow-parent/vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/gridConnector.js

```diff
@@ -810,17 +810,6 @@ import { isFocusable } from '@vaadin/grid/src/vaadin-grid-active-item-mixin.js';
           grid._cache.updateSize();
         });
 
-        const isSelectedOnGrid = function (item) {
-          const selectedItems = grid.selectedItems;
-          for (let i = 0; i < selectedItems; i++) {
```

Comment:
The loop condition `i < selectedItems` was flawed because `selectedItems` is an array, effectively never running the loop, making the function always return false.